comment_iowriter.go

```diff
@@ -18,7 +19,7 @@ func NewRawCommentWriter(w io.Writer) *RawCommentWriter {
 	return &RawCommentWriter{w: w}
 }
 
-func (s *RawCommentWriter) Post(c *Comment) error {
+func (s *RawCommentWriter) Post(_ context.Context, c *Comment) error {
```

Comment:
**[golint]** <sub>reported by [reviewdog](https://github.com/haya14busa/reviewdog) :dog:</sub>
exported method RawCommentWriter.Post should have comment or be unexported